main0.py

```diff
@@ -15,12 +16,13 @@
 import POGOProtos.Networking.Responses_pb2
 import POGOProtos.Networking.Requests
 import POGOProtos.Networking.Requests.Messages_pb2
-
+import pushbullet
+import os.path
```

Comment:
I took the code from the comments on your reddit page(with some customizing). I assumed it was for the people who installed the PB API directly.